consult.el

```diff
@@ -3569,6 +3572,11 @@ version supports preview of the selected string."
     (push-mark)
     (insert-for-yank string)
     (setq this-command 'yank)
+    (when yank-from-kill-ring-rotate
```

Comment:
yank-from-kill-ring-rotate probably doesn't exist on Emacs 27? I think we have to.introduce a variable consult-yank-rotate.

Reply:
You're right, I'll fix it

Reply:
On 28 the variable could be a defalias.

Reply:
I'm not sure how to do this though

Reply:
Maybe it's simpler to have the consult-specific variable have default value `(bound-and-true-p yank-from-kill-ring-rotate)`.

Reply:
This means that this feature will not work if emacs version is lower than 28

Reply:
Oh, I got confused I thought the rotation behavior was new in Emacs 28, but what's new is that you can turn it off? In that case, definitely go with your default and not my suggestion.